pypit/arutils.py

```diff
@@ -48,7 +48,7 @@ def bspline_inner_knots(all_knots):
     i1=pos[-1]
     return all_knots[i0:i1]
 
-
+# TODO JFH: This is the old bspline_fit which shoul be deprecated. I think some codes still use it though. We should transtion to pydl everywhere
```

Comment:
Raise this to an Issue.  A ToDo isn't enough.
I suggest Milestone = v0.9

Reply:
Done.